### PR TITLE
Add new examples

### PR DESCRIPTION
**Pull request by timwarner-msft**
Added examples to the `New-AzPolicyAssignment` and `New-AzPolicyDefinition` help files to show customers how to get the reader-friendly `DisplayName` property from those objects.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by timwarner-msft** [comment]
cc: @shanhix1, @mikefrobbins

File: src/Resources/Resources/help/Get-AzPolicyAssignment.md [hunk 1]
```diff
@@ -68,6 +68,15 @@ Get-AzPolicyAssignment -Scope '/providers/Microsoft.Management/managementgroups/
 The first command specifies the ID of the management group to query.
 The second command gets all of the policy assignments that are assigned to the management group with ID 'myManagementGroup'.
 
+### Example 4: Get the scope, policy set definition identifier, and display name of all policy assignments formatted as a list
+
+```powershell
+Get-AzPolicyAssignment | Select-Object -ExpandProperty properties | Select-Object Scope, PolicyDefinitionID, DisplayName | Format-List
```
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyAssignment.md:74]:
```suggestion
Get-AzPolicyAssignment | Select-Object -ExpandProperty properties | Select-Object -Property Scope, PolicyDefinitionID, DisplayName | Format-List
```

File: src/Resources/Resources/help/Get-AzPolicyDefinition.md [hunk 2]
```diff
@@ -84,10 +84,19 @@ This command gets all built-in policy definitions from the subscription with ID
 
 ### Example 5: Get policy definitions from a given category
 ```powershell
-Get-AzPolicyDefinition | Where-Object {$_.Properties.metadata.category -eq "Virtual Machine"}
+Get-AzPolicyDefinition | Where-Object {$_.Properties.metadata.category -eq "Tag"}
```
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyDefinition.md:87]:
```suggestion
Get-AzPolicyDefinition | Where-Object {$_.Properties.metadata.category -eq 'Tag'}
```

File: src/Resources/Resources/help/Get-AzPolicyDefinition.md [hunk 3]
```diff
@@ -84,10 +84,19 @@ This command gets all built-in policy definitions from the subscription with ID
...
 ```
 
-This command gets all policy definitions in category "Virtual Machine".
+This command gets all policy definitions in the category "Tags."
+
+### Example 6: Get the display name, description, policy type, and metadata of all policy definitions formatted as a list
+
+```powershell
+Get-AzPolicyDefinition | Select-Object -ExpandProperty properties | Select-Object DisplayName, Description, PolicyType, Metadata | Format-List
```
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyDefinition.md:95]:
```suggestion
Get-AzPolicyDefinition | Select-Object -ExpandProperty properties | Select-Object -Property DisplayName, Description, PolicyType, Metadata | Format-List
```

File: src/Resources/Resources/help/Get-AzPolicyAssignment.md [hunk 4]
```diff
@@ -68,6 +68,15 @@ Get-AzPolicyAssignment -Scope '/providers/Microsoft.Management/managementgroups/
...
+```
+
+This command is useful when you need to find the reader-friendly `DisplayName` property of an Azure
```
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyAssignment.md:77]:
```suggestion
This command is useful when you need to find the reader-friendly **DisplayName** property of an Azure
```

File: src/Resources/Resources/help/Get-AzPolicyDefinition.md [hunk 5]
```diff
@@ -84,10 +84,19 @@ This command gets all built-in policy definitions from the subscription with ID
...
+```
+
+This command is useful when you need to find the reader-friendly `DisplayName` property of an Azure
+Policy definition. You can parse the `Metadata` property to discover the policy definition's version number and category assignment.
```
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyDefinition.md:99]:
```suggestion
This command is useful when you need to find the reader-friendly **DisplayName** property of an Azure
Policy definition. You can parse the **Metadata** property to discover the policy definition's version number and category assignment.
```

File: src/Resources/Resources/help/Get-AzPolicyAssignment.md (hunk 4 above)
Comment (mikefrobbins) [inline src/Resources/Resources/help/Get-AzPolicyAssignment.md:77]:
```suggestion
This command is useful when you need to find the reader-friendly **DisplayName** property of an Azure
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
Use full parameter names instead of positional parameters. Couple of minor style issues, but otherwise looks good.

**Review by msJinLei — APPROVED** [review]

**Review by msJinLei — APPROVED** [review]

**Comment by msJinLei** [comment]
@mikefrobbins could you help to see whether all of your comments are addressed? thanks

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**